imap_processing/hi/hi_l2.py

```diff
@@ -441,4 +462,6 @@ def esa_energy_df(
     esa_energies_lut = pd.read_csv(
         esa_energies_path, comment="#", index_col="esa_energy_step"
     )
+    if esa_energy_steps is None:
+        return esa_energies_lut
```

Comment:
Another option to not have to special case None I think is to use `esa_energy_steps = slice(None)` upon entry. So in your code above set the slice().